<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdWindow</class>
 <widget class="QMainWindow" name="CmdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit"/>
        <widget class="QPushButton" name="pushButton_Enter">
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Enter</string>
         </property>
        </widget>
       </widget>
       <widget class="QTextEdit" name="textEdit"/>
      </widget>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="pushButton_Clear">
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Finish</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
